### DISCUSSION TITLE
MLRS performance problem, relate to their category

### DISCUSSION THREAD

**Discussion by Luigi-II** [PygmalionOfCyprus/cmo-db-requests#5429] [General]
I noticed that MLRS munitions in the database, such as M30/31, PrSM, ATACMS and BRE series, are categorized as [ballistic missile]. However, I think that creates some unreasonable situations. In the current game simulation, I noticed that they would fly up to 200km and hit the ground at an unreasonably high speed (4000 or even 5000 knots, totally different from the Speed Band writes). It is generally known that these rockets (not re-entry vehicles) do not have that high a ballistic apex (generally 30-40km?) and it won't be that fast. Because they do not need to withstand the high overload and temperature that RV and HGV have to, their shells tend to be not that strong, allowing the normal HE Frag SAMs to effectively kill them. But currently they can't be effectively killed by HE Frag SAMs, I must use my precious Kinetic-Kill ABM assets to intercept them effectively.

I wonder how the DB team should handle this? My suggestion is to re-categorize them as normal [Guided Weapon] though that would make them perform like some kind of lofted-trajectoy supersonic cruise missile.

If the Russians hadn't lied to us:
https://armyrecognition.com/focus-analysis-conflicts/army/conflicts-in-the-world/russia-ukraine-war-2022/russia-claims-to-intercept-10-us-atacms-missiles-over-crimea-with-tor-m2-air-defense-system-2
I'm not sure if the SA-15D has Hit-to-Kill capability, but for now I'm treating it as if it doesn't.

**Comment by Luigi-II** [comment]
#5417

**Comment by ddranidis** [comment]
We had a look at this.

We tested ATACMS at a max-range shot, and the apogee was just over 50km and terminal (at-impact) velocity was ~2300 kts or Mach ~4. These figures seem consistent with public information on this weapon:
![image](https://github.com/user-attachments/assets/3bca69f2-2818-4fa4-bb0f-e2ace75389c7)

We also tested PrSM at max range; this is a much longer-ranged missile, and both apogee and terminal velocity were much higher. This is again as expected.

The ballistic trajectory code appears to work as designed. Please be aware that less-than-max-range shots tend to raise the apogee even higher, as the trajectory has to compensate for the reduced range (you can observe this most clearly in NK missile tests).

We are aware that certain distinct systems (e.g. Iskander) are explicitly designed for depressed-trajectory flight, but this is a future model addition.
